command/state_meta.go

```diff
@@ -23,8 +25,9 @@ func (c *StateMeta) State(m *Meta) (state.State, error) {
 		return nil, err
 	}
 
+	env := c.Env()
```

Comment:
Oh, actually there is no more Meta.State with the new backends, so this structure probably isn't needed at all.